lib/tasks/example_data.rake

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "faker"
+
+namespace :example_data do
+  desc "Create personas, their providers and a selection of trainees"
+  task generate: :environment do
+    if Rails.env.production?
+      puts "THIS TASK CANNOT BE RUN IN PRODUCTION"
```

Comment:
should this raise exception in order to use exit code, since its a no op

Reply:
Could do but I thought the output was better without all the exception stuff.

Reply:
I was thinking devops....

It blows up in the build tasks

Reply:
It's probably ok for it just not to run but I've updated it to raise instead.